lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java

```diff
@@ -88,28 +94,65 @@ public ReportUserIdentityHandler() {
     public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context)
             throws IOException {
         LogHelper.attachComponentId(configService);
-
         LOGGER.info(LogHelper.buildLogMessage("Processing report."));
 
+        ReportProcessingResult.ReportProcessingResultBuilder reportProcessingResult =
+                ReportProcessingResult.builder();
         try {
-            var userIds =
-                    vcStoreItemDataStore.getItems().stream()
-                            .map(VcStoreItem::getUserId)
-                            .distinct()
-                            .toList();
-            List<ReportUserIdentityItem> reportUserIdentities =
-                    userIds.stream()
-                            .map(
-                                    usrId ->
-                                            new ReportUserIdentityItem(
-                                                    usrId,
-                                                    Vot.P0.name(),
-                                                    Collections.emptyList(),
-                                                    false))
-                            .toList();
-            reportUserIdentityDataStore.createOrUpdate(reportUserIdentities);
+            ReportProcessingRequest reportProcessingRequest =
+                    objectMapper.readValue(inputStream, ReportProcessingRequest.class);
+            LOGGER.info(
+                    LogHelper.buildLogMessage(
+                            "Request received-" + reportProcessingRequest.toString()));
+            List<String> userIds = new ArrayList<>();
+            if ((!reportProcessingRequest.continueUserScan()
+                            && (reportProcessingRequest.tacticalStoreLastEvaluatedKey() == null))
+                    || reportProcessingRequest.continueUserScan()) {
```

Comment:
Can be simplified to `reportProcessingRequest.continueUserScan() || reportProcessingRequest.tacticalStoreLastEvaluatedKey() == null`

But why wouldn't we supply `continueUserScan: true` even if there is no continuation key?

Reply:
Changed to just
 if (reportProcessingRequest.continueUniqueUserScan())